src/content/contributing/code-review/_index.en.md

```diff
@@ -6,44 +6,41 @@ weight: 30
 
 ## Code Review Guide
 
-This guide is meant to facilitate Submariner code review by sharing norms, best
-practices, and useful patterns.
+This guide is meant to facilitate Submariner code review by sharing norms, best practices, and useful patterns.
 
-Submariner follows the [Kubernetes Code Review Guide][kube code review guide]
-wherever relevant. This guide collects the most important highlights of the
-K8s process and adds Submariner-specific extensions.
+Submariner follows the [Kubernetes Code Review Guide][kube code review guide] wherever relevant. This guide collects the most important
+highlights of the K8s process and adds Submariner-specific extensions.
 
-### Two non-author approvals required
+### Two non-author Committer approvals required
 
-Pull Requests to Submariner require two non-author code review approvals.
-
-At least one approval must be from a Committer to the relevant part of the code
-base, as defined by the CODEOWNERS file at the root of the repository.
+Pull Requests to Submariner require two approvals from a Committer to the relevant part of the code base, as defined by the CODEOWNERS file
+at the root of each repository and the [Community
+Membership/Committers](http://localhost:1313/contributing/community-membership/#committers) process.
 
 ### No merge commits
 
 Kubernetes recommends [avoiding merge commits][merge commits].
 
-Use `git fetch` and `git rebase` to avoid them.
+With our current GitHub setup, pull requests are liable to include merge commits temporarily: whenever a PR is updated through the UI,
+GitHub merges the target branch into the PR. However, since we merge PRs by either squashing or rebasing them, those merge commits
+disappear from the series of commits which ultimately ends up in the target branch.
 
 ### Squash/amend commits into discrete steps
 
 Kubernetes recommends [squashing commits using these guidelines][squashing].
 
 After a review, prepare your PR for merging by squashing your commits.
 
-All commits left on your branch after a review should represent meaningful
-milestones or units of work. Use commits to add clarity to the development and
-review process.
+All commits left on your branch after a review should represent meaningful milestones or units of work. Use commits to add clarity to the
+development and review process.
```

Comment:
To further expand:

```suggestion
development and review process. Keep in mind that smaller commits are easier to review. In addition. when addressing review comments, as a general rule, push a new commit instead of amending to the prior commit as the former makes it easy for reviewers to determine what changed.
```

Reply:
Following on Tom’s suggestion: 

```
But remember to squash them into the appropriate commit before merging. To make that simpler, commit with `git commit --fixup`, with the appropriate commit hash: this will keep your review-prompted changes as separate commits, but automatically select them for squashing with `git rebase --autosquash -i`.
```